src/Employer/Employer.Web/ViewModels/Part1/Training/TrainingViewModel.cs

```diff
@@ -1,12 +1,11 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
 
 namespace Esfa.Recruit.Employer.Web.ViewModels.Part1.Training
 {
     public class TrainingViewModel : TrainingEditModel
     {
+        public int MinYear => DateTime.UtcNow.Year;

```

Comment:
I'd ignored doing this type of validation in the ViewModel as I think this is a business rule error not a data validation one. But in this case its helpful as we can limit the HTML control. 👍

Reply:
This is more of a hint helper. I am just limiting some of the html input fields in a light way. A user can still put 2015 for example. The helper just makes a difference if they are using the up/down arrows.